ink-wrapper-types/src/aleph_client.rs

```diff
@@ -32,6 +35,14 @@ enum Code {
     Existing(CodeHash),
 }
 
+#[derive(Encode)]
+pub struct CodeUploadRequest {
```

Comment:
Do I understand correctly that this matches the structure from the substrate API? If so, can we add a comment about it so that no one tries to change it.

Reply:
Yeah, will do.